### ISSUE TITLE
Web Service Aggregate is missing

### ISSUE THREAD

**Issue by PaulFurlet** [microsoft/ALAppExtensions#4360]
Where Web Service is gone?
Error AL0185: Table 'Web Service Aggregate' is missing
Error AL0118: The name '"Service Name"' does not exist in the current context
Error AL0118: The name '"Object Type"' does not exist in the current context
Error AL0118: The name '"Object ID"' does not exist in the current context
Error AL0118: The name '"Object Type"' does not exist in the current context
Error AL0118: The name '"Object ID"' does not exist in the current context
Error AL0118: The name '"Service Name"' does not exist in the current context
Error AL0118: The name 'Published' does not exist in the current context

**Comment by AndersLarsen65** [comment]
Thanks for reporting this issue we will investigate and return as soon as possible

**Comment by onbuyuka** [comment]
Table 9900 "Web Service Aggregate" is located in Modules/System/Web Service Management/src folder. If you have the latest "System Application" package then you should be able to access the table since it is marked as Access=Public.

**Comment by PaulFurlet** [comment]
Extension is installed on BC14 and recently got email about this from Microsoft, that's why asked for advice how to change. Email text below:

Dynamics 365 Business Central
Your extension needs updating
The XXX extension that is installed in the Dynamics 365 Business Central tenant for YYY is not compatible with version 15.0.36590. The compatibility issue must be resolved before the tenant for which the extension is installed can be successfully updated to version 15.0.36590.
The compatibility issue must be resolved before the tenant for which the extension is installed can be successfully updated to version 15.0.36590.
Extension details: 
Name: XXX
Version: 1.0.0.5
Publisher: ZZZ
ID: xxxxxxxxxxxxxxx
Validation results:
Tenant: YYY
Application Version:        15.0.36590
Errors:
Error AL0185: Table 'Web Service Aggregate' is missing
Error AL0185: Table 'Web Service Aggregate' is missing
Error AL0118: The name '"Service Name"' does not exist in the current context
Error AL0118: The name '"Object Type"' does not exist in the current context
Error AL0118: The name '"Object ID"' does not exist in the current context
Error AL0118: The name '"Object Type"' does not exist in the current context
Error AL0118: The name '"Object ID"' does not exist in the current context
Error AL0118: The name '"Service Name"' does not exist in the current context
Error AL0118: The name 'Published' does not exist in the current context                                                                                                                                                         Recommended  actions:
Option 1: Extension is required (you may need to contact your Partner to perform this action)
A. Review the breaking changes and how to resolve them information on GitHub. Make the           minor change to the extension, compile the extension against the latest symbols for the major and minor application version the validation was run against, and perform the necessary tests.                                          B.  Upload the extension using the Extension Management page in Business Central and select the appropriate version for which the updated extension is compatible. The extension will then be updated to this version immediately or during the next minor or major update. For more information, view Deploying a Tenant Customization.                                                                                            Important: If you are using the extension, you can continue using the extension on the current version for 90 days. After 90 days, you will be updated to the next release without the extension. Please ensure that you have resolved the extension compatibility issue described above prior to such date.
Option 2: Extension is not required
Uninstall the extension using the Extension Management page in Business Central prior to being updated to the next minor or major version.

**Comment by RobAnderson42** [comment]
Hi Paul. We've identified an issue in our automated per-tenant extension validation process where it was not using the current version of the system and application symbols to perform the compilation. As a result, the "Web Service Aggregate" table was reported as missing when it should not have been.

The issue has been corrected and we'll be doing another round of validations in the coming days. If you do not receive another email saying that your extension needs updating it is good to go.

Thank you for reporting the issue and I apologize for any inconvenience this issue has caused.
